src/internal.c

```diff
@@ -7257,6 +7257,10 @@ static int GetRecordHeader(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
             WOLFSSL_MSG("DTLS handshake, skip RH version number check");
         else {
             WOLFSSL_MSG("SSL version error");
+            /* send alert per RFC 5246 Section 7.2.1 */
+            if(ssl->options.side == WOLFSSL_CLIENT_END) {
```

Comment:
Why only client side?

Reply:
I do not know the definite answer to that other than the fact that we want to limit alerts sent from the server to prevent a scan attack.